src/screencast/pipewire_screencast.c

```diff
@@ -26,6 +26,38 @@ static void writeFrameData(void *pwFramePointer, void *wlrFramePointer,
 	return;
 }
 
+static struct spa_pod *build_format(struct spa_pod_builder *b, enum spa_video_format format,
+		uint32_t width, uint32_t height, uint32_t framerate) {
+	struct spa_pod_frame f[1];
+
+	enum spa_video_format format_without_alpha = xdpw_format_pw_strip_alpha(format);
+
+	spa_pod_builder_push_object(b, &f[0], SPA_TYPE_OBJECT_Format, SPA_PARAM_EnumFormat);
+	spa_pod_builder_add(b, SPA_FORMAT_mediaType, SPA_POD_Id(SPA_MEDIA_TYPE_video), 0);
+	spa_pod_builder_add(b, SPA_FORMAT_mediaSubtype, SPA_POD_Id(SPA_MEDIA_SUBTYPE_raw), 0);
+	/* format */
+	if (format_without_alpha == SPA_VIDEO_FORMAT_UNKNOWN) {
+		spa_pod_builder_prop(b, SPA_FORMAT_VIDEO_format, 0);
+		spa_pod_builder_id(b, format);
```

Comment:
Can this be simplified to the following?

```c
spa_pod_builder_add(b, SPA_FORMAT_VIDEO_format, SPA_POD_Id(format), 0);
```

Reply:
yes